src/components/FindMovie/FindMovie.tsx

```diff
@@ -14,44 +66,59 @@ export const FindMovie: React.FC = () => {
             <input
               data-cy="titleField"
               type="text"
+              value={query}
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              className={classNames('input', {
+                'is-danger': !isFound,
+              })}
+              onChange={(e) => handleInput(e.target.value)}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {!isFound && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
+              disabled={query.length === 0 && true}
               type="submit"
-              className="button is-light"
+              className={classNames('button is-light', {
+                'button is-loading': loaderState,
```

Comment:
```suggestion
                'is-loading': loaderState,
```
looks redundant here, cause you already wrote this class above

Reply:
done)